package-lock.json

```diff
@@ -30,22 +30,22 @@
       "integrity": "sha512-mkLq8nwaXmDtFmRkQ8ED/eA2CnVw4zr7dCztKalZXBvdK5EeNUAesrrwUqjQEzFgomJssayzB0aqlOsP1vGLqg==",
       "dev": true,
       "requires": {
-        "@babel/code-frame": "^7.10.4",
-        "@babel/generator": "^7.11.0",
-        "@babel/helper-module-transforms": "^7.11.0",
-        "@babel/helpers": "^7.10.4",
-        "@babel/parser": "^7.11.0",
-        "@babel/template": "^7.10.4",
-        "@babel/traverse": "^7.11.0",
-        "@babel/types": "^7.11.0",
-        "convert-source-map": "^1.7.0",
-        "debug": "^4.1.0",
-        "gensync": "^1.0.0-beta.1",
-        "json5": "^2.1.2",
-        "lodash": "^4.17.19",
-        "resolve": "^1.3.2",
-        "semver": "^5.4.1",
-        "source-map": "^0.5.0"
+        "@babel/code-frame": "7.10.4",
+        "@babel/generator": "7.11.0",
+        "@babel/helper-module-transforms": "7.11.0",
+        "@babel/helpers": "7.10.4",
+        "@babel/parser": "7.11.0",
+        "@babel/template": "7.10.4",
+        "@babel/traverse": "7.11.0",
+        "@babel/types": "7.11.0",
+        "convert-source-map": "1.7.0",
+        "debug": "4.1.1",
+        "gensync": "1.0.0-beta.1",
+        "json5": "2.1.3",
+        "lodash": "4.17.19",
```

Comment:
Note: this also resolves the [lodash security alert](https://github.com/department-of-veterans-affairs/vets-json-schema/network/alert/yarn.lock/lodash/open).